settings/command_list.json

```diff
@@ -136,5 +136,10 @@
             "apistatus"
         ],
         "description": "Prints the status of the Riot Games API"
+    },
+    "endpoint": {
+        "aliases": [
+            "endpoint"
```

Comment:
aliases for itself are not necessary.

Reply:
iirc

Reply:
The bot does not respond to `!endpoint status` if I remove it from aliases (just tested again - that's also the reason why I added it in the first place)